scripts/source-maintenance.sh

```diff
@@ -216,7 +244,25 @@ git grep "ifn?def .*_SQUID_" |
 #
 # Scan for file-specific actions
 #
-for FILENAME in `git ls-files`; do
+ForkPoint=""
+FilesToOperateOn=""
+if test "x$OnlyChangedSince" = "x" ; then
+    FilesToOperateOn=`git ls-files`
+fi
+if test "x$FilesToOperateOn" = "x" && test "x$OnlyChangedSince" = "xfork" ; then
```

Comment:
The first condition of this test cannot ever be false when the second is true. This case has an else-if relationship with the statement above.
```suggestion
elif test "x$OnlyChangedSince" = "xfork" ; then
```